src/app/fdctl/run/tiles/fd_quic.c

```diff
@@ -310,7 +310,7 @@ after_frag( void *             _ctx,
       return;
     }
 
-    if( FD_UNLIKELY( *opt_sz>FD_TPU_MTU ) ) {
+    if( FD_UNLIKELY( *opt_sz-network_hdr_sz>FD_TPU_MTU ) ) {
```

Comment:
Do you need to check for underflow?

Reply:
I think the check above it ensures it can't underflow